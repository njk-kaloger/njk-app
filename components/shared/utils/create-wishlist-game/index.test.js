import createWishlistGame from "./";

it("create a wishlist game", () => {
  expect(
    createWishlistGame(
      "The Witcher 3: Wild Hunt",
      "PC",
      "70",
      "RPG",
      "https://images-na.ssl-images-amazon.com/images/I/71NrauAMwbL._SY445_.jpg",
      "https://thumbs.gfycat.com/ExcitableLimitedBufeo-size_restricted.gif"
    )
  ).toMatchSnapshot();
});
