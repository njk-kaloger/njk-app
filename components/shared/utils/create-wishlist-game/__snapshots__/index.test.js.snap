// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a wishlist game 1`] = `
Object {
  "animatedImage": "https://thumbs.gfycat.com/ExcitableLimitedBufeo-size_restricted.gif",
  "genre": "RPG",
  "id": "PC-The Witcher 3: Wild Hunt",
  "image": "https://images-na.ssl-images-amazon.com/images/I/71NrauAMwbL._SY445_.jpg",
  "platform": "PC",
  "price": Array [],
  "title": "The Witcher 3: Wild Hunt",
}
`;
